src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -237,6 +239,22 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
             WorkspaceService.ListAllWorkspaces)
       }
     } ~
+    path("admin" / "publishedWorkspaces") {
+      get {
+        requestContext => perRequest(requestContext,
+          WorkspaceService.props(workspaceServiceConstructor, userInfo),
+          WorkspaceService.ListPublishedWorkspaces)
+      }
+    } ~
+    path("admin" / "workspacesByAttribute") {
+      post {
+        entity(as[AttributeMap]) { attributeMap =>
```

Comment:
I understand the problem. Semantically, though, this feels like it should be a GET. Also I feel like distinguishing the type in such a subtle way is not a great API. I know it is verbose, but what about query parameters:
attributeName
valueString
valueBoolean
valueNumber

it also seems icky but it is clear what is going on.

Alternately, I would love a way that queried independent of type (i.e. "123" would match the number 123 as well as the string).